.github/workflows/semantic.yml

```diff
@@ -29,6 +29,7 @@ jobs:
             revert
 
           scopes: |
+            http
```

Comment:
```suggestion
```

This change seems to affect the `http` and `axiom` sinks? Could we use those pre-existing scopes instead?

Reply:
I don't know :) I did this because it was one of the review comments https://github.com/vectordotdev/vector/pull/21583#issuecomment-2450699285 :)

Reply:
I thought it is worth introducing a scope since this affects a lower level module used by multiple components. But I don't really have a strong opinion here.

Reply:
Aha gotcha, sorry for the run-around 😅 Maybe `http provider` would be more consistent? We have other `x providers` already as scopes to use for lower level changes that affect multiple components that share a "provider".

Reply:
I removed this http component from list as suggested